boot/zephyr/include/os/os_heap.h

```diff
@@ -20,19 +20,34 @@
 #ifndef H_OS_HEAP_
 #define H_OS_HEAP_
 
+#ifdef __ZEPHYR__
+#include <zephyr.h>
+#else
 #include <stddef.h>
+#endif // __ZEPHYR__
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#ifdef __ZEPHYR__
+
+#define os_malloc k_malloc
```

Comment:
is this for mbedTLS or for TinyCBOR?

Reply:
This was for mbedTLS, thanks for point this because thanks to #150 it is unused code.

Reply:
loader.c fille also call malloc so I would keep these definitions.